examples/with-nextjs-and-clerk-auth/src/app/globals.css

```diff
@@ -2,4 +2,5 @@ body,
 html {
   margin: 0;
   padding: 0;
+  min-height: 100vh;
```

Comment:
I would prefer to not fix the layout (`min-height`), while it is possible. It's better to fix a wrapper.